rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
```

Comment:
The rental should be adding itself to the `@book` to achieve the **belongs-to** association. To achieve this update your code accordingly. Like this: `book.rentals << self`.